app/src/main/assets/cinnabar-style-7.0.1.yaml

```diff
@@ -718,8 +718,12 @@ textures:
 
 sources:
     mapzen:
-        type: MVT
-        url:  http://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.mvt
+        type: TopoJSON #GeoJSON, MVT, TopoJSON
+        url:  https://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.topojson
+        #url: '//tile.dev.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.topojson'
+        #url:  //localhost:8080/vector/all/{z}/{x}/{y}.json
```

Comment:
Yes, it accepts any of these formats. But the binary MVT is smaller, this makes a difference with this many users.